hw/top_earlgrey/data/ip/chip_clkmgr_testplan.hjson

```diff
@@ -188,8 +188,13 @@
             jitter_o clkmgr output to toggle. Verify this output is connected to AST's
             clk_src_sys_jen_i input using formal.
 
+            Test the following properties of jitter enable:
+            - The reset value of jitter_enable is off.
+            - Any write to jitter_enable turns jitter on.
+            - jitter_regwen has no impact on jitter_enable.
```

Comment:
I think testing that the regwen has no impact is fine but what we really care about is that we cannot disable it again once it's set to enable and this is currently not done. I think it would be valuable to add this.

Reply:
Ah, actually we cannot test this anymore as the dif in it's reworked form doesn't allow this. This should be alright then or what is your view @matutem ?

Reply:
I added a test for this in clkmgr_jitter_test.c: first clear regwen and then set jitter enable and check that updates to enable go through. The dif code doesn't matter for this test.

Reply:
By the way, I think jitter regwen getting stuck once cleared should be a unit test. There is a clkmgr_regwen_vseq.sv test that should have done this, but it only checks extclk_regwen and meas_ctrl_regwen, darn! I am pretty bummed about this bug!

I will add this check in that unit test in a separate PR. I think we can commit this separately, since it enables Nuvoton to check the jitter_enable fix at top-level.

Reply:
A units test for jitter_regwen was added in #26259.